platforms/ide/problems/src/main/kotlin/org/gradle/problems/internal/impl/DefaultProblemsReportCreator.kt

```diff
@@ -127,9 +125,12 @@ class JsonProblemWriter(private val problem: Problem, private val failureDecorat
                         )
                     }
                 }
+                problem.contextualLabel?.let {
+                    property("contextualLabel", it)
+                }
                 problem.definition.documentationLink?.let { property("documentationLink", it.url) }
                 problem.exception?.let { writeError(failureDecorator.decorate(failureFactory.create(it))) }
-                property("category") {
+                property("group") {
```

Comment:
❓Shouldn't we group by ID? ie. the property name should be ID and the list should contain `DefaultProblemId`s.

Reply:
this refers to the same thing we decided to call "group" [here](https://github.com/gradle/configuration-cache-report/pull/26#discussion_r1745115032)